sources/packages/api/src/services/bceid/bceid.service.ts

```diff
@@ -0,0 +1,152 @@
+import { Injectable } from "@nestjs/common";
+import { BasicAuthSecurity, Client, createClientAsync } from "soap";
+import { InjectLogger } from "../../common";
+import { BCeIDConfig } from "../../types/config";
+import { LoggerService } from "../../logger/logger.service";
+import { ConfigService } from "../config/config.service";
+import { AccountDetails } from "./account-details.model";
+import {
+  BCeIDAccountTypeCodes,
+  ResponseBase,
+  ResponseCodes,
+} from "./bceid.models";
+
+/**
+ * Manage the execution of SOAP requests to the BCeID Web Service as per described
+ * on https://sminfo.gov.bc.ca/ based on the documentation available on
+ * "BCeID Client - New Web Services - Developers Guide - V2.10.0 (for BCeID WS V10).docx"
+ */
+@Injectable()
+export class BCeIDService {
+  private bceidConfig: BCeIDConfig;
+  constructor(private readonly config: ConfigService) {
+    this.bceidConfig = this.config.getConfig().bceid;
+  }
+
+  /**
+   * This method allows you to retrieve the details of a single BCeID account.
+   * This method respects the information sharing rules. To perform a query, there must be an
+   * information sharing rule for the online service (onlineServiceId) with the specified requester
+   * user account type (requesterAccountTypeCode) and target user account type (accountTypeCode).
```

Comment:
i'd put this as 
"user account type (requesterAccountTypeCode - Business/Internal)"
same for target user account type

Reply:
Both variables are using an Enum that makes it clear what are the possible values to be used. The enum items also have a comment on each to explain the idea behind it. I think that move the option from the enum and adding it into the parenthesis is not needed in this case.